custom/conf/app.ini.sample

```diff
@@ -869,6 +869,14 @@ SHOW_FOOTER_VERSION = true
 ; Show template execution time in the footer
 SHOW_FOOTER_TEMPLATE_LOAD_TIME = true
 
+[markup.sanitizer]
+; All three of the following keys are necessary to define one sanitation
+; policy rule and the set may be redefined multiple times to add multiple
+; rules.
+;ELEMENT = span
+;ALLOW_ATTR = class
+;REGEXP = ^\s*((math(\s+|$)|inline(\s+|$)|display(\s+|$)))+
```

Comment:
I think we should not use such a complex example here. I know it's exactly your use case, but it's too complex. How about?
```suggestion
;REGEXP = ^(info|warning|error)$
```
I think that it delivers the meaning and it's easier to understand. The more complex example can go in the cheat sheet.